api/room/schema.py

```diff
@@ -114,22 +132,27 @@ class Arguments:
         image_url = graphene.String()
         calendar_id = graphene.String()
         cancellation_duration = graphene.Int()
+        room_tags = graphene.List(graphene.Int)
     room = graphene.Field(Room)
 
     @Auth.user_roles('Admin')
     def mutate(self, info, room_id, **kwargs):
         validate_empty_fields(**kwargs)
         query_room = Room.get_query(info)
         active_rooms = query_room.filter(RoomModel.state == "active")
-        exact_room = active_rooms.filter(RoomModel.id == room_id).first()
-        if not exact_room:
+        room = active_rooms.filter(RoomModel.id == room_id).first()
+        if not room:
             raise GraphQLError("Room not found")
 
         admin_roles.update_delete_rooms_create_resource(room_id)
-        update_entity_fields(exact_room, **kwargs)
-
-        exact_room.save()
-        return UpdateRoom(room=exact_room)
+        room_tags = ''
+        if kwargs.get('room_tags'):
+            room_tags = kwargs.pop('room_tags')
+        update_entity_fields(room, **kwargs)
+        new_tags = room.room_tags
+        new_tags.clear()
```

Comment:
Kindly share the purpose of the .clear() method

Reply:
@RonKbS, the relationship table room_tags comes as a list, so when I'm editing the tags in a room, say a room has tags [1, 3, 4] and I want to update the tags to [1, 3], I chose to clear the list using the python method .clear() and append the new tags.